docs/source/pillars/information_governance.md

```diff
@@ -86,6 +86,12 @@ _The ability of the TRE organisation to monitor compliance with internal and ext
 * - You report on and share outcomes of each audit of your TRE organisation with the required bodies
   - This may included regulatory bodies or the organisations that manage accreditations you have
   - Mandatory
+* - You log user actions, network connection attempts, infrastructure actions and data access within a TRE
+  - The greater the number of (and diversity of) actions logged, the higher the chance that any bad behaviours, intentional or unintentional, are spotted.
+    Bad behaviours in this context are anything that could compromise the security of the SRE, or result in the accidental discolure of data outside the TRE.
+    Any measures that are valuable for reporting should be collected, for example, the number of times that unauthorised access to the TRE was blocked.
```

Comment:
Can we tell that from the logs?

For example, how do you distinguish someone trying to guess a password and someone mistyping a password?

Reply:
Is it useful even without being able to tell the difference? Can we think of a better example? I'm not sure what we log for DSH?

Reply:
If you can't tell the difference, you can't report how many failed access attempts with unauthorised. However, the text says that is what you should collect.

Reply:
I've watered down the wording here, but as a result I've downgraded the suggestion to Recommended